examples/release_highlights/plot_release_highlights_1_2_0.py

```diff
@@ -120,10 +120,13 @@
 # Improved efficiency of many estimators
 # --------------------------------------
 # In version 1.1 the efficiency of many estimators relying on the computation of
-# pairwise distances was greatly improved for float64 dense input. In version 1.2,
-# the efficiency of these estimators was further improved for all combinations of
-# float32/float64 and dense/sparse input (for all metrics except euclidean). It
-# concerns essentially clustering, manifold learning and neighbor search algorithms.
+# pairwise distances (essentially estimators related to clustering, manifold
+# learning and neighbors search algorithms) was greatly improved for float64
+# dense input. Efficiency improvement especially were a reduced memory footprint
+# a much better scalability on multi-core machines.
+# In version 1.2, the efficiency of these estimators was further improved for all
+# combinations of dense and CSR inputs on float32 and float64 datasets, except
+# the CSR-dense and dense-CSR combinations for the Euclidean and Squared Euclidean
```

Comment:
I tend to prefer sparse, even if less precise, because not all users know the acronym. And since it's the highlights I think we need to use a more high level language